src/Mutation/Mutation.php

```diff
@@ -142,10 +141,9 @@ public function getMutatedNode(): MutatedNode
         return $this->mutatedNode;
     }
 
-    // TODO: hasTest()?
     public function isCoveredByTest(): bool
     {
-        return $this->coveredByTests;
+        return $this->tests !== [];
```

Comment:
this looks like an unrelated change and the behaviour is different: previously this value was memoized, now it will be re-evaluated on every call